ios/FluentUI/Command Bar/MultilineCommandBar.swift

```diff
@@ -0,0 +1,151 @@
+        }
+        sheetController.preferredExpandedContentHeight = rowsStackView.systemLayoutSizeFitting(UIView.layoutFittingCompressedSize).height
+
+        NSLayoutConstraint.activate([
+            sheetController.view.topAnchor.constraint(equalTo: view.topAnchor),
+            sheetController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            sheetController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            sheetController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ])
+
+        bottomSheetController = sheetController
+    }
+
+    public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        if previousTraitCollection?.horizontalSizeClass != traitCollection.horizontalSizeClass || previousTraitCollection?.verticalSizeClass != traitCollection.verticalSizeClass {
+            removeRows()
+            if traitCollection.horizontalSizeClass == traitCollection.verticalSizeClass {
+                addRows(rows: self.regularRows)
+            } else {
+                addRows(rows: self.compactRows)
+            }
+            bottomSheetController?.preferredExpandedContentHeight = rowsStackView.systemLayoutSizeFitting(UIView.layoutFittingCompressedSize).height
+            updateCommandBarAppearance()
+        }
+    }
+
+    @available(*, unavailable)
+    public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    public var commandBarOverrideTokens: [CommandBarTokenSet.Tokens: ControlTokenValue]? {
+        didSet {
+            updateCommandBarAppearance()
+        }
+    }
+
+    // MARK: - Private
+
+    private var rowsStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    private var compactRows: [MultilineCommandBarRow]
+
+    private var regularRows: [MultilineCommandBarRow]
+
+    private var commandBarRows = [CommandBar]()
+
+    private var bottomSheetController: BottomSheetController?
+
+    private struct Constants {
+        static let horizontalPadding: CGFloat = GlobalTokens.spacing(.size160)
+    }
+
+    private func addRows(rows: [MultilineCommandBarRow]) {
+        for row in rows {
+            let commandBarRow = CommandBar(itemGroups: row.itemGroups, leadingItemGroups: nil)
+            commandBarRow.isScrollable = row.isScrollable
+            commandBarRow.translatesAutoresizingMaskIntoConstraints = false
+            commandBarRows.append(commandBarRow)
+
+            if row.isScrollable {
+                commandBarRow.tokenSet[.itemBackgroundColorRest] = .uiColor {
+                    UIColor(light: GlobalTokens.neutralColor(.white),
+                            dark: GlobalTokens.neutralColor(.black))
+                }
```

Comment:
We should probably expose these colors somewhere, or use alias tokens, or something. Just always setting the scrolling rows to white/black probably isn't what we want to do.